### DISCUSSION TITLE
What's the purpose of this project

### DISCUSSION THREAD

**Discussion by ralfeus** [tendermint/flutter#301] [General]
I didn't quite understand - what is the purpose of the project? Is it a client for Tendermint that connects to Tendermint Core or attempt to run Tendermint Core on Android? I couldn't find any description

**Comment by andrzejchm** [comment]
Hi @ralfeus, it's a repository of a work that's still in progress, but the general idea here is to:

1. host the code for a "starport template" - a flutter project with a template of a mobile app that allows you to access cosmos-based chain, and build features around it, some examples might be a mobile client for a DEX, or an Akash client for mobile

2. host code of libraries that allow you to streamline the process of building mobile apps that communicate with Cosmos-based chains, like:
 - `transaction_signing_gateway` - library for signing and broadcasting transactions on the chain
 - `cosmos_auth` - eases the process of storing and retrieving credentials in a secure way, so that your mobile app can store, for example, passwords or a private key securely to be able to retrieve it with biometrics for the user's convenience.
 
 We'll work on the documentation definitely as soon as we lay down the final shape of the repository and the libraries reach the stable state
